change/@azure-communication-react-624831d8-54c5-496d-90c3-d9fe253b9582.json

```
@@ -0,0 +1,9 @@
+{
+  "type": "patch",
+  "area": "fix",
+  "workstream": "Lazy loading",
+  "comment": "Fix lazy loading for RichTextSendBox",
```

It might be a good idea to change this to be "fix to composites" (as RTESendBox doesn't use lazy loading)
```suggestion
  "comment": "Fix lazy loading for Composites",
```